data/behavior_data.c

```diff
@@ -4478,7 +4483,7 @@ const BehaviorScript bhvBigBoulder[] = {
     OR_INT(oFlags, (OBJ_FLAG_COMPUTE_ANGLE_TO_MARIO | OBJ_FLAG_SET_FACE_YAW_TO_MOVE_YAW | OBJ_FLAG_UPDATE_GFX_POS_AND_ANGLE)),
     SET_FLOAT(oGraphYOffset, 180),
     CALL_NATIVE(bhv_big_boulder_init),
-    SET_FLOAT(oCollisionDistance, 20000),
+    SET_FLOAT(oDrawingDistance, 20000),
```

Comment:
why did you remove collision distance?

Reply:
oCollisionDistance is unused for Boulders

Reply:
oCollisionDistance was unused for boulders

Reply:
how so?

Reply:
i think this may need 20k so it can still hurt enemies right?

Reply:
Only objects that use load_object_collision_model are affected by oCollisionDistance. Boulders use set_rolling_sphere_hitbox for their collision.